diem-move/diem-framework/experimental/sources/NFT.move

```diff
@@ -48,8 +48,18 @@ address 0x1 {
             amount: u64,
         }
 
+        struct TransferEvent has copy, drop, store {
+            id: GUID::ID,
+            from: address,
+            to: address,
+            amount: u64,
```

Comment:
This we might have to think about later if order matters, but we don't have a unique identifier for each individual token (within a GUID).

Reply:
I hope it's ok: the whole idea of multi-token NFTs is that they're supposed to be fungible within the same GUID.